src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -101,7 +102,10 @@ object WorkflowDescriptor {
     validateWorkflowDescriptor(id, sourceFiles, CromwellBackend.backend(), conf) match {
       case scalaz.Success(w) => w
       case scalaz.Failure(f) =>
-        throw new IllegalArgumentException(s"""Workflow $id failed to process inputs:\n${f.toList.mkString("\n")}""")
+        throw new IllegalArgumentException() with ThrowableWithErrors {
+          val message = s"Workflow input processing failed."
```

Comment:
I realize this never makes it to the DB and therefore shouldn't go back in an error message as that was a bug we already fixed.  :smile:  But I'm wondering if the message that will be produced here will give enough context to be tied back to the submission.